### ISSUE TITLE
Every instance of monitor keeps connecting and disconnecting to MQTT broker

### ISSUE THREAD

**Issue by gieljnssns** [andrewjfreyer/monitor#67]
192.168.0.55  -->  v0.1.746
192.168.0.57  -->  v0.1.711
192.168.0.60  -->  v0.1.746
Now I'm using this broker https://github.com/hassio-addons/addon-mqtt
But I saw the same with this add-on https://www.home-assistant.io/addons/mosquitto/

```
1543305176: New client connected from 192.168.0.55 as Living2327 (c1, k60, u'hass').
1543305176: New connection from 192.168.0.57 on port 1883.
1543305176: New client connected from 192.168.0.57 as mosqpub|20411-PiZero-Ke (c1, k60, u'hass').
1543305176: Client mosqpub|20411-PiZero-Ke disconnected.
1543305176: Client Living2327 disconnected.
1543305176: New connection from 192.168.0.60 on port 1883.
1543305176: New client connected from 192.168.0.60 as Garage29313 (c1, k60, u'hass').
1543305176: Client Garage29313 disconnected.
1543305176: New connection from 192.168.0.60 on port 1883.
1543305176: New client connected from 192.168.0.60 as Garage29318 (c1, k60, u'hass').
1543305176: New connection from 192.168.0.55 on port 1883.
1543305176: New client connected from 192.168.0.55 as Living2338 (c1, k60, u'hass').
1543305176: Client Living2338 disconnected.
1543305176: Client Garage29318 disconnected.
1543305178: New connection from 192.168.0.57 on port 1883.
1543305178: New client connected from 192.168.0.57 as mosqpub|20478-PiZero-Ke (c1, k60, u'hass').
1543305178: Client mosqpub|20478-PiZero-Ke disconnected.
1543305178: New connection from 192.168.0.57 on port 1883.
1543305178: New client connected from 192.168.0.57 as mosqpub|20483-PiZero-Ke (c1, k60, u'hass').
1543305178: Client mosqpub|20483-PiZero-Ke disconnected.
1543305180: New connection from 192.168.0.57 on port 1883.
1543305180: New connection from 192.168.0.57 on port 1883.
1543305180: New client connected from 192.168.0.57 as mosqpub|20493-PiZero-Ke (c1, k60, u'hass').
1543305180: New client connected from 192.168.0.57 as mosqpub|20494-PiZero-Ke (c1, k60, u'hass').
1543305180: Client mosqpub|20493-PiZero-Ke disconnected.
1543305181: Client mosqpub|20494-PiZero-Ke disconnected.
1543305181: New connection from 192.168.0.57 on port 1883.
1543305181: New connection from 192.168.0.57 on port 1883.
1543305181: New client connected from 192.168.0.57 as mosqpub|20502-PiZero-Ke (c1, k60, u'hass').
1543305181: New client connected from 192.168.0.57 as mosqpub|20505-PiZero-Ke (c1, k60, u'hass').
1543305183: Client mosqpub|20502-PiZero-Ke disconnected.
1543305183: Client mosqpub|20505-PiZero-Ke disconnected.
1543305185: New connection from 192.168.0.55 on port 1883.
1543305185: New client connected from 192.168.0.55 as Living2538 (c1, k60, u'hass').
1543305185: New connection from 192.168.0.60 on port 1883.
1543305185: New client connected from 192.168.0.60 as Garage29412 (c1, k60, u'hass').
1543305185: Client Living2538 disconnected.
1543305185: New connection from 192.168.0.55 on port 1883.
1543305185: New client connected from 192.168.0.55 as Living2543 (c1, k60, u'hass').
1543305185: Client Garage29412 disconnected.
1543305185: Client Living2543 disconnected.
1543305185: New connection from 192.168.0.60 on port 1883.
1543305185: New client connected from 192.168.0.60 as Garage29417 (c1, k60, u'hass').
1543305185: Client Garage29417 disconnected.
1543305190: New connection from 192.168.0.55 on port 1883.
1543305190: New client connected from 192.168.0.55 as Living2583 (c1, k60, u'hass').
1543305190: Client Living2583 disconnected.
1543305190: New connection from 192.168.0.60 on port 1883.
1543305190: New client connected from 192.168.0.60 as Garage29490 (c1, k60, u'hass').
1543305190: Client Garage29490 disconnected.
1543305190: New connection from 192.168.0.55 on port 1883.
1543305190: New client connected from 192.168.0.55 as Living2588 (c1, k60, u'hass').
1543305190: Client Living2588 disconnected.
1543305190: New connection from 192.168.0.60 on port 1883.
1543305190: New client connected from 192.168.0.60 as Garage29495 (c1, k60, u'hass').
1543305190: Client Garage29495 disconnected.
1543305193: New connection from 192.168.0.55 on port 1883.
1543305193: New client connected from 192.168.0.55 as Living2619 (c1, k60, u'hass').
1543305194: Client Living2619 disconnected.
1543305194: New connection from 192.168.0.55 on port 1883.
1543305194: New client connected from 192.168.0.55 as Living2632 (c1, k60, u'hass').
1543305194: Client Living2632 disconnected.
1543305195: New connection from 192.168.0.57 on port 1883.
1543305195: New client connected from 192.168.0.57 as mosqpub|20694-PiZero-Ke (c1, k60, u'hass').
1543305195: Client mosqpub|20694-PiZero-Ke disconnected.
1543305195: New connection from 192.168.0.57 on port 1883.
1543305195: New client connected from 192.168.0.57 as mosqpub|20699-PiZero-Ke (c1, k60, u'hass').
1543305195: Client mosqpub|20699-PiZero-Ke disconnected.
1543305196: New connection from 192.168.0.55 on port 1883.
1543305196: New client connected from 192.168.0.55 as Living2767 (c1, k60, u'hass').
1543305196: Client Living2767 disconnected.
1543305196: New connection from 192.168.0.55 on port 1883.
1543305196: New client connected from 192.168.0.55 as Living2772 (c1, k60, u'hass').
1543305196: Client Living2772 disconnected.
1543305198: New connection from 192.168.0.55 on port 1883.
1543305198: New client connected from 192.168.0.55 as Living2780 (c1, k60, u'hass').
1543305198: Client Living2780 disconnected.
1543305199: New connection from 192.168.0.55 on port 1883.
1543305199: New client connected from 192.168.0.55 as Living2785 (c1, k60, u'hass').
1543305199: Client Living2785 disconnected.
1543305210: New connection from 192.168.0.57 on port 1883.
1543305210: New client connected from 192.168.0.57 as mosqpub|20713-PiZero-Ke (c1, k60, u'hass').
1543305210: Client mosqpub|20713-PiZero-Ke disconnected.
1543305210: New connection from 192.168.0.57 on port 1883.
1543305210: New client connected from 192.168.0.57 as mosqpub|20718-PiZero-Ke (c1, k60, u'hass').
1543305210: Client mosqpub|20718-PiZero-Ke disconnected.
1543305214: New connection from 192.168.0.57 on port 1883.
1543305214: New client connected from 192.168.0.57 as mosqpub|20743-PiZero-Ke (c1, k60, u'hass').
1543305214: Client mosqpub|20743-PiZero-Ke disconnected.
1543305214: New connection from 192.168.0.57 on port 1883.
1543305214: New client connected from 192.168.0.57 as mosqpub|20748-PiZero-Ke (c1, k60, u'hass').
1543305214: Client mosqpub|20748-PiZero-Ke disconnected.
1543305215: New connection from 192.168.0.57 on port 1883.
1543305215: New client connected from 192.168.0.57 as mosqpub|20815-PiZero-Ke (c1, k60, u'hass').
1543305216: Client mosqpub|20815-PiZero-Ke disconnected.
1543305216: New connection from 192.168.0.57 on port 1883.
1543305216: New client connected from 192.168.0.57 as mosqpub|20820-PiZero-Ke (c1, k60, u'hass').
1543305216: Client mosqpub|20820-PiZero-Ke disconnected.
1543305218: New connection from 192.168.0.57 on port 1883.
1543305218: New client connected from 192.168.0.57 as mosqpub|20836-PiZero-Ke (c1, k60, u'hass').
```

**Comment by andrewjfreyer** [comment]
This is the nature of mosquitto_pub, at least as far as the man page for the bin is concerned. Are you noticing any negative impacts?

**Comment by xbmcnut** [comment]
I had the same issue and this activity flooded my broker so bad, every other MQTT device on my network ground to a halt. If you are using the inbuilt Home Assistant MQTT, it may be an compatibility issue between the two? From memory, when I used a docker MQTT instance on my NAS, it settled down but in the end, I decommissioned the system completely as debugging it went in the too hard bin.

**Comment by xbmcnut** [comment]
@andrewjfreyer All I can tell you as when I had two instances of presence or monitor running on my network and connected to the Home Assistant MQTT broker, all of my other devices became unresponsive. I ended up pulling out units, reflashing firmware and all sorts to try and debug it. When I saw the constant connect/disconnect messages, I took both monitor units offline and everything came back to normal.
